pkgs/development/python-modules/slither-analyzer/default.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  makeWrapper,
+  prettytable,
+  pythonOlder,
+  solc,
+}:
+
+buildPythonPackage rec {
+  pname = "slither-analyzer";
+  version = "0.3.0";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "10vrcqm371kqmf702xmqmzimv3xgrn3k3ip06nr1l6gnj3jk138g";
+  };
+
+  buildInputs = [ makeWrapper ];
```

Comment:
```suggestion
  nativeBuildInputs = [ makeWrapper ];
```

Reply:
Out of curiosity: how come? The [java manual](https://nixos.org/nixpkgs/manual/#sec-language-java) uses `buildInputs`, same as other packages.

Reply:
Because it's a build time only dependency.